src/ngraph_encapsulate_op.cc

```diff
@@ -36,6 +36,10 @@
 
 #include "ngraph/runtime/interpreter/int_backend.hpp"
 
+#ifdef NGRAPH_DISTRIBUTED
```

Comment:
It's better to use the `if defined(NGRAPH_DISTRIBUTED)`

Reply:
Changed to `#if defined NGRAPH_DISTRIBUTED`